core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -796,14 +768,9 @@ public int getIndex() {
             return index;
         }
 
-        boolean validState() { return state == State.VALID_STATE;}
 
-        public static <K, V> IteratorState<K, V> endState() {return endState;}
-        public static <K, V> IteratorState<K, V> state(Chunk<K, V> nextChunk, Chunk.ChunkIter nextChunkIter, int nextIndex) {
-            return new IteratorState<>(nextChunk, nextChunkIter, nextIndex, State.VALID_STATE);
-        }
         public static <K, V> IteratorState<K, V> initState(Chunk<K, V> nextChunk, Chunk.ChunkIter nextChunkIter) {
-            return new IteratorState<>(nextChunk, nextChunkIter, Chunk.NONE, State.INIT_STATE);
+            return new IteratorState<>(nextChunk, nextChunkIter, Chunk.NONE);
```

Comment:
Perhaps capitalize this method or rename it to emphasize this is a static factory method and not an ordinary method. Static factory methods [often](https://docs.oracle.com/javase/tutorial/datetime/overview/naming.html) start with `from` or `of`.